tooling/cli/src/dev.rs

```diff
@@ -78,6 +78,21 @@ pub fn command(options: Options) -> Result<()> {
 fn command_internal(options: Options) -> Result<()> {
   let logger = Logger::new("tauri:dev");
 
+  #[cfg(not(debug_assertions))]
+  match check_for_updates() {
+    Ok((msg, sleep)) => {
+      if sleep {
+        logger.log(msg);
+        std::thread::sleep(std::time::Duration::from_secs(3));
```

Comment:
@amrbashir it is to make user aware of the update, if we don't sleep at that time next command might clear the terminal buffer and user might miss it completely or not care about it.

Reply:
but that delay is bad for UX IMO

Reply:
what can be an alternative, may be ask for auto update (Y/n) instead?

Reply:
No, that also is a bad UX. I think we should just remove this delay. And if users don't see the notification, it is not that important. 

Also note that after stable 2.0 release, users might deliberately choose to stay on 1.x for stability and we probably shouldn't annoy them because of that.

I honestly think we should remove this notification after 1.0 stable release is out because the CLI and other Tauri components will be stable by then and there will be no breaking changes moving forwards so you wouldn't need to update the CLI as often as the "beta" or "rc" phase.